root/ntuple/ntuple-makeproject/makeproject_ttree.cxx

```diff
@@ -0,0 +1,7 @@
+#include <TFile.h>
+
+int main()
+{
+   std::unique_ptr<TFile> f{TFile::Open("ntuple_makeproject_stl_example_ttree.root")};
+   f->MakeProject("libttreestltest", "*", "recreate++");
```

Comment:
```suggestion
   if (!f || f->IsZombie())
      return 1;
   f->MakeProject("libttreestltest", "*", "recreate++");
   return 0;
```

Reply:
Why is the `return 0` necessary?

Reply:
To ensure that the test executable return success/zero.